base/db/R/insert.format.vars.R

```diff
@@ -1,28 +1,67 @@
 #' Insert Format and Format-Variable Records 
-#' @param con Bety connection object
+#'
+#' @param con SQL connection to BETYdb
+#' @param format_notes Character string that describes format.
+#' @param format_name Character string 
+#' @param header Boolean that indicates the presence of a header in the format 
+#' @param skip Integer that indicates the number of lines to skip in the header. Defaults to "" 
+#' @param mimetype_id Integer 
 #' @param formats_variables_df A 'data.frame' consisting of entries that correspond to columns in the formats-variables table
-#' @param formats_df A 'data.frame' consisting of entries that correspond to columns in the formats table
+#' @details It is very important that the formats_variables_df 'data.frame' be structured in a specific format so that the SQL query functions properly. First, all arguments should be passed as vectors so that each entry will correspond with a specific row.
+#' Variables: 
+#' \enumerate{
+#' \item variable_id: vector of integers
+#' \item name: vector of character strings
+#' \item unit: vector of character strings
+#' \item storage_type: vector of character strings
+#' \item column_number: vector of integers
+#' }
 #' @author Liam Burke (liam.burke24@gmail.com)
-#' @return Data frame: Inner join of SQL table and input data frame (as unevaluated "lazy query" table) 
+#' @return format_id
 #' @export
 #' @examples
 #' \dontrun{
 #' bety <- betyConnect
-#'   insert.format.vars(con = bety$con, formats_df, formats_variables_df)
+#' 
+#' formats_variables_df <- tibble::tibble(
+#'        variable_id = c(327, 2.98e+02), # integer 
+#'        name = c("New Name", "Other New Name"), # 
+#'        unit = c("Yoonits", "Other Units"),
```

Comment:
This is a really bad example because:
1. variable 327 (`c2n_root`) is unlikely to be in a column named "New Name" and "Yoonits" is most definitely not convertible by udunits2 to the standard units for C:N ratios. Data on C:N ratios are also not going to be used to store the day of year (`%j`)
2. variable numbers should not be entered using scientific notation. Given that variable 298 is LE (latent heat flux), the name, units, and storage type entered are just as absurd as the first column.

Reply:
I apologize for the confusion. This was a placeholder example since I had not made a formal formats-variable record with a real data set at that point. I have now done this and the new example is updated and should address these points.